Sources/JWT/AtomicJwks.swift

```diff
@@ -0,0 +1,85 @@
+import Vapor
+
+/// A thread-safe class for retrieving JSON Web Key Sets which honors the
+/// HTTP `Cache-Control`, `Expires` and `Etag` headers.
+final public class AtomicJwks {
```

Comment:
The standard for camel casing acronyms is if the acronym appears after the first word in the name, it should be all upper-case. So `AtomicJwks` should actually be `AtomicJWKS`.

Reply:
I updated the name, thanks.